api/organisations/urls.py

```diff
@@ -133,7 +172,7 @@
     )
 
     nested_roles_router = routers.NestedSimpleRouter(
-        organisations_router, r"roles", lookup="role"
+        organisations_router, parent_prefix=r"roles", lookup="role"
```

Comment:
Why are we updating roles router here?

Reply:
It seems to be the result of a merge with some branch, corrected.